apps/src/labs/header/Lab2ShareDialog.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Wrapper around ShareDialog that plumbs in the necessary props for a Lab2 project.
+ */
```

Comment:
minor style question, do we have a preference of putting top-level docs after or before imports in functional react components? I think I end up putting them after imports, on top of the component declaration, but could go either way

Reply:
I don't have a strong preference here, I can change to after imports

Reply:
I think one advantage of having them before imports is that something after imports might apply to the entire file, or it might just apply to the topmost of several separate things in the file, and it's hard to differentiate.

Reply:
I'm fine with either, but I agree having at the very top can be slightly clearer.

Reply:
That's fair, I think my reasoning for having them after imports is that if especially if there are a lot of imports, it can help the reader visually skip the imports and go directly to where the content of the file start (especially if there's only a single component in the file). But agreed on having them at the top when there are multiple distinct things in a file. Fine either way!

Reply:
merged with comment below imports, agree we can put them at the top if there are multiple functions/classes in a single file